en/mkdocs.yml

```diff
@@ -595,8 +595,12 @@ nav:
                   - AmazonS3 Reference: reference/connectors/amazons3-connector/1.x/amazons3-connector-1.x-reference.md
           - Amazon SQS Connector:
               - Amazon SQS Overview: reference/connectors/amazonsqs-connector/amazonsqs-connector-overview.md
-              - Amazon SQS Example: reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md
-              - Amazon SQS Reference: reference/connectors/amazonsqs-connector/amazonsqs-connector-config.md
+              - 2.x:
```

Comment:
As discussed, please use the same format as for version 1.x and create a separate directory for version 2.x content. Also, rename the file to include '2.x' in the name.

Additionally, add redirects for the two URLs we will no longer be using to prevent 404 errors when switching between versions.